tracing/providers/jaeger/src/main/java/io/helidon/tracing/providers/jaeger/JaegerTracerProvider.java

```diff
@@ -29,6 +33,10 @@
  */
 @Weight(Weighted.DEFAULT_WEIGHT)
 public class JaegerTracerProvider implements TracerProvider {
+
+    private static final List<SpanListener> LIFE_CYCLE_LISTENERS = HelidonServiceLoader.create(ServiceLoader.load(
```

Comment:
Nit: lifecycle is one word, so `LIFECYCLE_LISTENERS`, or, better yet, given the rename, `SPANLISTENERS`

Reply:
Done